data-prepper-plugins/otel-proto-common/src/main/java/org/opensearch/dataprepper/plugins/otel/codec/OTelLogsFormatOption.java

```diff
@@ -15,6 +15,7 @@
 
 public enum OTelLogsFormatOption {
     JSON("json"),
+    STANDARD_JSON("standard_json"),
```

Comment:
I understand that the current `format` option is instructing the _input_ format into the codec. That is, how do we decode.

But, in the difference between "standard" and "opensearch" we are directing the _output_ from the codec.

I tend to think we should make this a new option instead of overloading this.